tests/image_creation.bats

```diff
@@ -45,5 +45,9 @@ function build_image()
 }
 
 @test "Can create euleros image" {
+	if [ "$TRAVIS" = true ]
```

Comment:
Hi @liangchenye - we might want to do this at some pont. However, this PR should just be for updating the README imho.

Reply:
I'll revert it and add a new PR..